dev-qt/qtwebengine/qtwebengine-6.9999.ebuild

```diff
@@ -53,7 +53,10 @@ RDEPEND="
 	media-libs/tiff:=
 	sys-apps/dbus
 	sys-apps/pciutils
-	sys-devel/gcc:*
+	|| (
+		sys-devel/gcc:*
+		llvm-runtimes/libatomic-stub
+	)
```

Comment:
While here, could you move this block to DEPEND at same time? This doesn't use the shared version of atomic that I can see and can be DEPEND-only. stub is also static-only anyway.

Can move it between `${RDEPEND}` and libglvnd dependency, thanks.